src/openzaak/components/documenten/api/viewsets.py

```diff
@@ -431,6 +440,107 @@ def _zoek(self, request, *args, **kwargs):
     _zoek.is_search_action = True
 
 
+@extend_schema_view(
+    create=extend_schema(
+        operation_id="enkelvoudiginformatieobject_import_create",
+        summary=_("Een IMPORT creeren"),
+        description=_(
+            "Creeert een IMPORT. Wanneer er vervolgens een metadata bestand"
+            " wordt aangeleverd zal de daadwerkelijke IMPORT van start gaan."
+        ),
+        request=OpenApiTypes.NONE,
+        responses={
+            (status.HTTP_201_CREATED, "application/json"): {
+                "type": "object",
+                "properties": {
+                    "uploadUrl": {
+                        "type": "string",
+                        "description": _("Url waar het metadata bestand geupload kan worden."),
+                    },
+                    "statusUrl": {
+                        "type": "string",
+                        "description": _("Url waar de status van de IMPORT opgevraagd kan worden."),
+                    },
+                    "reportUrl": {
+                        "type": "string",
+                        "description": _("Url waar het reportage bestand gedownload kan worden."),
+                    },
+                },
+                "example": {
+                    "uploadUrl": "https://example.com",
+                    "statusUrl": "https://example.com",
+                    "reportUrl": "https://example.com",
+                },
+            },
+            **COMMON_ERROR_RESPONSES,
+        },
+    )
+)
+class EnkelvoudigInformatieObjectImportView(ImportCreateview):
```

Comment:
all import endpoints should not be available if CMIS is enabled. Please take a look at Verzending viewset, which is also not available for CMIS